types/canvas-confetti/index.d.ts

```diff
@@ -81,6 +81,14 @@ declare namespace confetti {
          */
         y?: number;
     }
+    interface globalOpts {
+        resize: boolean;
+    }
+
+    function create(
+        canvas: HTMLCanvasElement,
+        options?: globalOpts
+    ): (options?: Options) => Promise<null> | null;
```

Comment:
Looks like it's actually a [`Promise<void>`](https://github.com/catdad/canvas-confetti/blob/master/src/confetti.js#L190) ([when it's not `null`](https://github.com/catdad/canvas-confetti/blob/master/src/confetti.js#L45)), but that's also a problem with the top level `function confetti`